argo_workflow_tools/argo_client.py

```diff
@@ -62,13 +65,16 @@ def submit(
             annotations (dict, optional): workflow annotations. Defaults to {}.
             labels (dict, optional): workflow labels. Defaults to {}.
             wait (bool, optional): block program and wait for workflow to finish. Defaults to False.
+            resource_kind (str, optional): resource kind to submit. Defaults to "WorkflowTemplate".
 
         Returns:
             WorkflowResult: workflow status reference
         """
 
         if namespace is None:
             namespace = self._options.namespace
+        if resource_kind.value not in (choices := WorkflowType.choices()):
```

Comment:
@8ball030 please notice that the project support python3.7+ (the walrus operator was added in 3.8)

Reply:
Ahh good eyes, I will fix this.